corehq/apps/locations/permissions.py

```diff
@@ -147,3 +129,43 @@ def can_edit_form_location(domain, web_user, form):
         if not form_location:
             return False
         return user_can_edit_location(web_user, form_location, domain_obj)
+
+
+#### Unified location permissions below this point
+# TODO incorporate the above stuff into the new system
```

Comment:
This is unfortunate, but the ad-hoc stuff from before is somewhat difficult to incorporate into this new system, since projects are using it without the user role.  My plan is to get feature parity and migrate over projects using the flags, then drop the old way of doing things.